src/main/java/xyz/duncanruns/julti/Julti.java

```diff
@@ -182,6 +197,8 @@ public void run() throws IOException {
         if (isRanFromAlternateLocation()) {
             log(Level.INFO, "Julti is being ran from another location");
         }
+        isWallAHKOpen();
```

Comment:
no error handling is done here, which adds another failure point for something which isn't important for Julti being able to run